src/providers/consensus/client.py

```diff
@@ -33,9 +34,33 @@ class ConsensusClient(HTTPProvider):
     API_GET_BLOCK_HEADER = '/eth/v1/beacon/headers/{}'
     API_GET_BLOCK_DETAILS = 'eth/v2/beacon/blocks/{}'
     API_GET_VALIDATORS = 'eth/v1/beacon/states/{}/validators'
+    _API_GET_CONSENSUS_SPEC = 'eth/v1/config/spec'
```

Comment:
`_API_GET_CONSENSUS_SPEC` -> `API_GET_CONSENSUS_SPEC`?

Reply:
it isn't  provided that this method and its endpoint will be called directly from outside, so it is a private attribute of the class